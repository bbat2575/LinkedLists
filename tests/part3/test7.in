NEW 1
1
NEW 1
2
INSERT 1 0 {0}
NEW 4
1
2.2
$
bazinga
VIEW-NESTED 2
TYPE 2
INSERT 2 4 {1}
INSERT 2 -1 {1}
INSERT 2 0 {1}
INSERT 2 -5 {1}
VIEW-NESTED 2
TYPE 2
INSERT 2 0 {1}
NEW 1
1
INSERT 3 0 {1}
INSERT 3 1 {1}
INSERT 3 -1 {1}
INSERT 3 -2 {1}
VIEW-NESTED 3
TYPE 3
