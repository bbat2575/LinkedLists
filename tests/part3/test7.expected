
Type HELP for a list of options
Press <CTRL + D> to Quit

> List 0: 1

> List 1: 2

> Nested 1: {List 0} -> 2

> List 2: 1 -> 2.20 -> $ -> bazinga

> 1 -> 2.20 -> $ -> bazinga

> int -> float -> char -> string

> INVALID COMMAND: INSERT

> INVALID COMMAND: INSERT

> INVALID COMMAND: INSERT

> INVALID COMMAND: INSERT

> 1 -> 2.20 -> $ -> bazinga

> int -> float -> char -> string

> INVALID COMMAND: INSERT

> List 3: 1

> INVALID COMMAND: INSERT

> INVALID COMMAND: INSERT

> INVALID COMMAND: INSERT

> INVALID COMMAND: INSERT

> 1

> int

> 
